ubuntu-gcc-devel/Dockerfile

```diff
@@ -11,9 +11,13 @@ RUN cd /tmp \
 
 ENV RPREFIX /opt/R-devel
 
-ENV ROPTIONS --with-recommended-packages=no --enable-R-shlib --enable-R-static-lib
+ENV ROPTIONS --with-recommended-packages --enable-R-shlib --enable-R-static-lib
+
+RUN apt-get update \
+    && apt-get -y install rsync
 
 RUN cd /tmp/R-devel \
+    && ./tools/rsync-recommended \
     && R_PAPERSIZE=letter                                    \
     R_BATCHSAVE="--no-save --no-restore"                     \
     PERL=/usr/bin/perl                                       \
```

Comment:
I tried building and running this one too, similarly building apparently went well but then R could not be found.

Reply:
it is in `/opt`